flake.nix

```diff
@@ -474,37 +519,44 @@
                         "https://0x98650451ba02064f7b000f5768cf0cf4d4e492317d82871bdc87ef841a0743f69f0f1eea11168503240ac35d101c9135@mainnet-relay.securerpc.com"
                         "https://0xa1559ace749633b997cb3fdacffb890aeebdb0f5a3b6aaa7eeeaf1a38af0a8fe88b9e4b1f61f236d2e64d95733327a62@relay.ultrasound.money"
                       ]} \
-                      -addr 0.0.0.0:18550
+                      -addr ${cfg.${activeConsensusClient}.mev-boost.endpoint}
```

Comment:
Later on we might reiterate this -- strictly speaking, MEV-boost is not part of lighthouse. There could be a separate MEV-boost option, which if enabled, get's enabled to each ConsensusClient. This way, if the ConsensusClient is enabled, MEV-boost would also automatically be enabled.